schedule/yast/lvm_raid1/lvm+raid1_sle15_default.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
we don´t need to call them `_default.yaml` but use the architecture/backend used.
Later we will refactor to have only 1 file without extension.
For the extension, better going with underscore i.e.: `_arch_backend.yaml`